docs/sources/reference/commandline/cli.md

```diff
@@ -427,7 +427,6 @@ Docker supports softlinks for the Docker data directory
     export DOCKER_TMPDIR=/mnt/disk2/tmp
     /usr/local/bin/docker -d -D -g /var/lib/docker -H unix:// > /var/lib/boot2docker/docker.log 2>&1
 
-
```

Comment:
Hmm, I think when I removed the text that I added I removed the extra line.  Added it back in.